test/genbank_download_test.py

```diff
@@ -164,28 +162,3 @@ def test_check_for_taxonomy_bug(self):
 
 
 
-
```

Comment:
Was this removed because it was effectively a duplicate?  Or was this a previous attempt?